### DISCUSSION TITLE
How does one discern HOW a site is blocking?

### DISCUSSION THREAD

**Discussion by dwhempy** [ultrafunkamsterdam/undetected-chromedriver#1152] [General]
How does one go about determining how a site is preventing access? I've been to httpbin.org and looked through the data there but (and I'll admit to a shallow understanding of things) I'm not seeing anything obvious when I compare manually opening the site versus using Python/Selenium/UC. I'm hoping that knowing which blocking method will lead to some sort of method to get around it.

I am trying to access a financial site so I expect it to be difficult. I'm just looking to login to my account, grab a screenshot, and e-mail it. Python 3.9 and Chrome 111.

**Comment by pulth** [comment]
I want to know this too. I'm currently being detected by https://prenotami.esteri.it/ after log in and don't know why. I'm new to this package so any ideas are welcome.

Following image shows the error I get after trying an automated log in:
![image](https://user-images.githubusercontent.com/76142591/227390866-b1405d71-b2b2-4520-8419-e580f930da17.png)
